src/olympia/reviewers/tests/test_views.py

```diff
@@ -8779,3 +8780,83 @@ def test_bigquery_disabled(self, get_adu_per_version_mock):
 
         assert response.status_code == 503
         assert response.json() == {}
+
+    def test_review_page_html(self):
+        version = self.addon.current_version
+        review_url = reverse('reviewers.review', args=[self.addon.pk])
+        response = self.client.get(review_url)
+        assert response.status_code == 200
+        doc = pq(response.content)
+        # the version id as the anchor
+        assert doc(f'#version-{to_dom_id(version.version)}.listing-header')
+        assert doc('.version-adu').attr('data-version-string') == version.version
+        # url to the usage_per_version view, and what the max results will be
+        addon_div = doc('#addon')
+        assert addon_div.attr('data-versions-adu-url') == self.url
+        assert addon_div.attr('data-versions-adu-max-results') == str(VERSION_ADU_LIMIT)
+        # url to the review_version_redirect
+        assert addon_div.attr('data-review-version-url') == reverse(
+            'reviewers.review_version_redirect', args=(self.addon.id, '__')
+        )
+
+
+class TestReviewVersionRedirect(ReviewerTest):
+    def login_as_reviewer(self):
+        self.grant_permission(
+            UserProfile.objects.get(email='reviewer@mozilla.com'),
+            'Addons:ReviewUnlisted',
+        )
+        return super().login_as_reviewer()
+
+    def test_responses(self):
+        addon = addon_factory()
+        listed = addon.current_version
+        unlisted = version_factory(addon=addon, channel=amo.CHANNEL_UNLISTED)
+        deleted = version_factory(addon=addon)
+        deleted.delete()
```

Comment:
Add another add-on to make sure we only take the requested add-on into consideration